src/main/java/com/pinterest/secor/io/impl/AvroParquetFileReaderWriterFactory.java

```diff
@@ -8,8 +8,8 @@
 import com.pinterest.secor.common.SecorSchemaRegistryClient;
 import org.apache.avro.Schema;
 import org.apache.avro.generic.GenericRecord;
-import org.apache.avro.io.Encoder;
-import org.apache.avro.io.EncoderFactory;
+import org.apache.avro.io.*;
```

Comment:
Don't use * in the import, please list each individual classes

Reply:
Fixed